case-lib/lib.sh

```diff
@@ -672,6 +672,31 @@ is_zephyr()
     test "$znum" -gt 10
 }
 
+is_firmware_file_zephyr()
+{
+    local firmware_name firmware_path znum platform
+    # get the FW name and path
+    # TODO: optimize this part after driver can expose the FW path to userspace via debugfs
+    if is_ipc4; then
+        firmware_name=dsp_basefw.bin
+        fw_mod_para=/sys/module/snd_sof_pci/parameters/fw_path
+        if [ -s "$fw_mod_para" ]; then
+            firmware_path=$(cat $fw_mod_para)
+        else
+            # TODO: let the kernel driver expose the FW path
+            # and get the FW path by grepping journalctl.
+            return 1
```

Comment:
We should `die` here because we don't know the answer. `return 1` means "not Zephyr"

Reply:
Updated with die.